src/vivarium_census_prl_synth_pop/data/nicknames_proportion.py

```diff
@@ -0,0 +1,20 @@
+import pandas as pd
+from vivarium import Artifact
+
+from vivarium_census_prl_synth_pop.constants import data_keys, paths
+
+
+def get_nicknames_proportion():
+    # This function calculates the proportion of names that have potential nicknames. This will return a float that
+    # that will be used as a constant to scale the nicknames noise function in the pseudopeople packages.
+
+    art = Artifact(paths.DEFAULT_ARTIFACT)
+    names = art.load(data_keys.SYNTHETIC_DATA.FIRST_NAMES).reset_index()["name"]
```

Comment:
This ignores the "freq" column which is what makes some names more common than others. I think we want to correct this.